Sources/Apodini/Semantic Model Builder/REST/RESTDefaultRootHandler.swift

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Andreas Bauer on 22.01.21.
+//
+
+@_implementationOnly import Vapor
+
+/// A RoutesHandler which is automatically registered to the root path
+/// if there is no Endpoint registered under the root, in order to server entry point links.
+struct RESTDefaultRootHandler {
+    let configuration: RESTConfiguration
+    let relationships: Set<RelationshipDestination>
+
+    // Registers a GET handler on root path
+    func register(on app: Vapor.Application) {
+        app.get(use: self.handleRequest)
+    }
+
+    func handleRequest(_: Vapor.Request) -> ResponseContainer {
+        let links = relationships.formatRelationships(into: [:], with: LinksFormatter(configuration: configuration))
```

Comment:
nit: Could we somehow configure the relationships formatter from outside? That would be really cool if people could inject their own RelationshipFormatter's

Reply:
Definitively. But would call this out of scope of this PR.
I will create an issue for it?